ammo-be/src/utils/ValidationUtil/__tests__/test.ts

```diff
@@ -0,0 +1,28 @@
+import { isValidUrl, hasValidWeight } from 'utils/ValidationUtil';
+
+describe('ValidationUtil', () => {
+    describe('URL validation regex', () => {
```

Comment:
Why is there two describes ?

Reply:
It's an error, I replaced the 2nd decribe with the tested method name : 
```js
describe('ValidationUtil', () => {
    describe('isValidUrl', () => {
```